.gitignore

```diff
@@ -23,3 +23,5 @@ package-lock.json
 *.njsproj
 *.sln
 *.sw?
+src/Components/Header.css
+src/Components/Header.jsx
```

Comment:
why two components are in the .gitignore? 🧐

Reply:
I can´t remember.  🤔